### PR TITLE
Replacing Nightwatch tests with Wdio tests- Overlay

### PR DESCRIPTION
**Pull request by paku-sruti**
### Summary
Replacing Nightwatch tests with Wdio tests- Overlay.
[JIRA](https://jira2.cerner.com/browse/CWELL-16150)

### CODE REVIEW

File: packages/terra-overlay/tests/wdio/overlay-spec.js [hunk 1]
```diff
@@ -1,6 +1,163 @@
...
+
+  describe('Scrollable relative container overlay', () => {
+    beforeEach(() => browser.url('/#/tests/overlay-tests/container-scrollable'));
+
+    Terra.should.beAccessible();
+    Terra.should.matchScreenshot();
+  });
+
+  describe('Overlay on Request-Close', () => {
+    describe('OnRequestClose- Full Screen Overlay', () => {
```
Comment (emilyrohrbough) [inline packages/terra-overlay/tests/wdio/overlay-spec.js]: These tests are nicely written good job!! My only request, can we get a before & after screenshot of the `on request close` overlays?

File: packages/terra-overlay/tests/wdio/overlay-spec.js [hunk 2]
```diff
@@ -1,6 +1,163 @@
...
 
 describe('Overlay', () => {
+  describe('Default', () => {
+    beforeEach(() => browser.url('/#/tests/overlay-tests/default'));
+
+    Terra.should.beAccessible({ selector: '#default-overlay' });
+    Terra.should.matchScreenshot({ selector: '#default-overlay' });
+
+    it('Expect default to have full screen', () => {
+      expect(browser.getAttribute('#default-overlay', 'class')).contains('fullscreen');
```
Comment (dm123455) [inline packages/terra-overlay/tests/wdio/overlay-spec.js]: I would say if we have screen shots depicting things like full screen etc, we do not have to write explicit wdio tests for it. If it is something that cannot be captured in a screenshot like spin animation/scrollability then add tests for it.

File: packages/terra-overlay/tests/wdio/overlay-spec.js (hunk 1 above)
Comment (paku-sruti) [reply packages/terra-overlay/tests/wdio/overlay-spec.js]: Thanks @emilyrohrbough  :). Updated the test cases.

**Comment by paku-sruti** [comment]
@bjankord This PR can be merged after you approve it.

**Pull request merged**